content/en/docs/reference/scheduling/policies.md

```diff
@@ -11,99 +11,13 @@ that the {{< glossary_tooltip text="kube-scheduler" term_id="kube-scheduler" >}}
 runs to [filter and score nodes](/docs/concepts/scheduling-eviction/kube-scheduler/#kube-scheduler-implementation),
 respectively.
 
-You can set a scheduling policy by running
-`kube-scheduler --policy-config-file <filename>` or
-`kube-scheduler --policy-configmap <ConfigMap>`
-and using the [Policy type](/docs/reference/config-api/kube-scheduler-policy-config.v1/).
```

Comment:
Maybe we should leave this text in, or something like it, to make it clear what the thing is that we're telling readers they shouldn't use.
eg
```markdown
In Kubernetes versions before v1.23, you were able to set a scheduling policy by
passing command line argumets to the kube-scheduler. For example, you might
have configured the scheduler to run with the command line
`kube-scheduler --policy-config-file my-policy.yaml`.
``` 

If so, we could omit the previous paragraph.

Reply:
Explain what it is also seems meaningful.